CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v3.0.1
+
+### Map
+
+* Fixed double finger tap gesture zoom out position.
+* Added the support for automatically changing the map style to dark when entering tunnels.
+
+### Banners and guidance instructions
+
+* Added support for the road shied name in the `WayNameView`.
+* Fixed double road name setting.
+
+### Other changes
+
+* Optimized `NavigationRoutes` initialization.
+* Fixed `NavigationMapView` DocC.
+* Fixed rare incorrect route line displaying after rerouting.
```

Comment:
```suggestion
* Fixed `NavigationMapView` DocC.
* Fixed a rare incorrect route line displaying after rerouting.
```
Would it make sense to move these two lines to the Map section?

Reply:
Absolutely! Thank you.